clients/drcachesim/tests/invariant_checker_test.cpp

```diff
@@ -43,6 +43,8 @@
 #include "../tools/invariant_checker.h"
```

Comment:
Title:

> i#6662: invariant_checker update

For multi-commit issues we use a very short title after the issue number (see https://dynamorio.org/page_code_reviews.html#sec_commit_messages), sometimes with ", part N": so something like `i#6662 regdeps` for the prefix.  Plus I think you could fit what the update is about: so I would suggest this title: `i#6662 regdeps: Invariant checks for filetype and cpuid`.

Reply:
Agreed, much better description of just "update". Done.